### DISCUSSION TITLE
There a discord server for this mod?

### DISCUSSION THREAD

**Discussion by Kersakof** [TEHE-Studios/ExpandedHelicopterEvents#349] [Q&A]
Not looking for help for the mod specifically, but I came across a thread directing me to this mod in terms of commissioning a map. Couldn't find any info on it after googling and searching forums, so instead of DMing I thought I'd just poke here since I'm anti-social af.

**Comment by Chuckleberry-Finn** [comment]
There is no discord server for this mod. There is an unofficial discord server for the modding community that does have a section meant for requests and commissions.

Here is the invite link:
https://discord.com/invite/SReMnbV4V7
